pkg/api/types.go

```diff
@@ -1733,6 +1733,44 @@ type NodeStatus struct {
 	Images []ContainerImage `json:"images,omitempty"`
 }
 
+// AvoidPods desribes pods that should avoid this node.
```

Comment:
Add: "This is the value for a Node annotation with key scheduler.alpha.kubernetes.io/preferAvoidPods and will eventually become a field of NodeStatus."

Also, although the multi-scheduler annotation doesn't have validation, I think AvoidPods should, because it is fairly complex. (Multi-scheduler annotation value is just a string, whereas value here is a JSON object.) You can look at how @kevin-wangzefeng added validation for node affinity: trace the code starting at the call to ValidateAffinityInPodAnnotations() from ValidatePodSpecificAnnotations() in pkg/api/validation/validation.go (you will need to add a function ValidateNodeSpecificAnnotations() since one does not exist yet, and call it from ValidateNode(); you can just follow the pattern that was used for pod). I think the only thing you need to validate is that exactly one field of PodSignature is set, so this should be easy. (Obviously only zero or one fields of PodSignature can be set today, but it will be good to have this validation code later if we add more fields.)